emanifest-py/pyproject.toml

```diff
@@ -1,6 +1,36 @@
+[project]
+name = "emanifest"
+version = "2.0.7"
```

Comment:
In the next fix, I'd like to see if we can auto-increment the package number. I'm going to create a new issue for actions work like this.

Reply:
Very cool, curious to see how we can do this.